charts/superset/values.yaml

```diff
@@ -47,7 +47,9 @@ serviceAnnotations: {}
 
 podAnnotations: {}
 
-securityContext: {}
+securityContext:
+  fsGroup: 1000
```

Comment:
Could you explain why this is set to 1000? I was trying to get my head around the security context stuff but from what I was reading from the docs. I'mm guessing you tested this with your dibs on thursday and it worked but I was hoping I could figure out why and I couldn't quite explain it myself

(https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) it reads to me like fsGroup is "normally" set to `2000` and that we also ought to the set `runAsUser` and `runAsGroup`